src/Value.cs

```diff
@@ -209,18 +209,26 @@ public static Value FromObject(object? o, ValueKind kind)
                 switch (kind)
                 {
                     case ValueKind.Int32:
+                        if (o is null)
+                            throw new WasmtimeException($"The value `{o ?? "null"}` is not valid for WebAssembly type {kind}.");
```

Comment:
If `o` is `null`, do we need the `{o ?? "null"}` in the string interpolation? 

Perhaps simply `A null value is not valid for WebAssembly type {kind}.`?

Same comment here as for the rest of the string interpolations in this function.

Reply:
Looks like the tests need updating as well.